packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -2120,41 +2149,21 @@ describe('src/cy/commands/actions/click', () => {
         cy.get('.badge-multi').click()
       })
 
-      it('throws when subject is not in the document', (done) => {
```

Comment:
Why was this test removed? I'd have expected the new error message to be verified here to suggest .click().click() should be broken up.